src/components/Expander/Expander.reducers.js

```diff
@@ -0,0 +1,6 @@
+export function onExpand(state = {}, isExpanded) {
+	return {
+		...state,
+		isExpanded
+	};
+};
```

Comment:
May want to rethink the name/behavior of this reducer/handler. The name `onExpand` implies that `isExpanded` will always be `true` when it returns, which is not the case, currently.

I suggest you either:

1. add an `onCollapse` handler/reducer which is called if `isExpanded` is true
2. change the name to `onToggle` or something more appropriate to the behavior